net/rsync/Makefile

```
@@ -30,8 +30,8 @@ define Package/rsync
   SECTION:=net
   CATEGORY:=Network
   SUBMENU:=File Transfer
-  TITLE:=Fast remote file copy program (like rcp)
-  DEPENDS:=+libpopt +zlib +RSYNC_xattr:libattr +RSYNC_acl:libacl +RSYNC_zstd:libzstd $(ICONV_DEPENDS)
+  TITLE:=rsync is an open source utility that provides fast incremental file transfer
```

```suggestion
  TITLE:=an open source utility that provides fast incremental file transfer
```

this title might be too long